code/modules/projectiles/updated_projectiles/gun_system.dm

```diff
@@ -946,6 +946,10 @@ and you're good to go.
 			projectile_to_fire.def_zone = user.zone_selected
 			if(C.stagger)
 				gun_scatter += 30
+		if(user.get_item_by_slot(SLOT_HEAD))
+			gun_accuracy_mult += user.get_item_by_slot(SLOT_HEAD).accuracy_mod
```

Comment:
this is failing the linter since it can cause runtimes

Reply:
I now know what you mean. Jones the cat, who can be equipped to the head slot, causes runtimes.

How would you suggest getting around this without adding accuracy_mod to the item parent?

Reply:
```
var/obj/item/clothing/head/head_slot = user.get_item_by_slot(SLOT_HEAD)
if(istype(head_slot))
    gun_accuracy_mult += head_slot.accuracy_mod
else
    gun_accuracy_mult += 0.20

Reply:
Nice, this works perfectly.